README.md

```diff
@@ -1,46 +1,27 @@
 # Aam Digital Setup
 This repository describes how to set up everything that is needed to run Aam Digital in production.
-This includes deploying the app, deploying and connecting the database and optionally deploying and connecting the permission backend.
+This includes deploying the app, deploying and connecting the database and optionally deploying and connecting the permission backend and keycloak.
 
-## Deploying without permission backend (simple)
-To set up the application with a direct connection to the database, without a permission backend, follow these steps:
+## Deploying the application
+The interactive script `interactive_setup.sh` walks you through the process of setting up the application.
+Just run it and look for the console output and questions.
 
-1. Clone this repository to easily get all the setup files: `git clone https://github.com/Aam-Digital/ndb-setup.git`
```

Comment:
are you still supposed to clone the setup repo or just download and use the interactive_setup.sh script? Let's keep very foolproof instructions here ;-)